lib/generator.go

```diff
@@ -435,14 +466,16 @@ func randomChefClientRun(config *Config, chefClient chef.Client, nodeName string
 	}
 
 	// TODO: (@afiune) Notify Data Collector of compliance report
```

Comment:
This PR doesn't cover this section of the code? Are we going to use it at all? if not, can we remove it?

Reply:
I did not take it out for now because what i'd really like to do with this generator is to harmonize it better with the rest of the way the generation stuff works with chef_runs.  When do that then that commented out section of code would be useful once again.  This pr just adds in generator, exactly as it works in our current ruby version, just so that we could stop using the ruby version of generator and get all things generation into chef_load (where they all belong).

Reply:
🌮